int8 header
int8 base_arg
int8 int_1
int8 int_2
int8 int_3
float32 float_1
float32 float_2
float32 float_3
